qutebrowser/browser/commands.py

```diff
@@ -692,9 +692,16 @@ def yank(self, what='url', sel=False, keep=False):
             flags = QUrl.RemovePassword
             if what != 'pretty-url':
                 flags |= QUrl.FullyEncoded
-            s = self._current_url().toString(flags)
+            current_url = self._current_url()
```

Comment:
You can just call this `url` - and please do `url = QUrl(self.current_url())` so we're actually getting a copy of the URL object. Otherwise we're manipulating the URL of the page being shown, and I have no idea what funny stuff could happen when doing so.